pkg/daemon/daemon_test.go

```diff
@@ -149,7 +149,7 @@ var testCases = []TestCase{
 	{
 		log:                         "ts2phc[1896327.319]: [ts2phc.0.config] dev/ptp4  offset    -1 s2 freq      -2",
 		MessageTag:                  "[ts2phc.0.config]",
-		Name:                        "ts2phc",
+		Name:                        "ts2phc2",
```

Comment:
name is process name right ?

Reply:
If you want me to change it back for the sake of making back ports easier let me know.